common/Vectorize.py

```diff
@@ -0,0 +1,167 @@
+"""Module to obtain Graph embeddings of a list of model graphs"""
+
+from google.cloud import spanner
+from karateclub import Graph2Vec
+import networkx
+import time
+
+class Vectorize:
+    """ Embedding class to create graph embeddings
+
+    Creates embeddings from graph structures of models.
+    """
+
+    _EDGE_ATTRS = [
+        "tensor_shape", "tensor_type"
```

Comment:
Are there any limitation for edge attributes and node attributes? If yes, can you add comment about the same here.

Reply:
Can only be attributes of Node/Edge class, added a comment for the same.